scripts/deploy.sh

```diff
@@ -6,19 +6,23 @@
 # a generate docker-compose file. Then it prunes older docker images, pulls
 # the latest docker image and restarts docker-compose.
 #
-PRODUCTION_SERVER="ubuntu@production.ldsolutions.org"
+# We have two enrivonments: production and staging. Please specify which host target
+# you wish to deploy when executing the script. For example, to deploy to production, 
+# execute "deploy.sh production".
 
-until bash -c "ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'docker ps'"; do
+SERVER="ubuntu@$1.ldsolutions.org"
```

Comment:
Can you add a check making sure that 1 argument is passed in when the script is called? If not we should just exit with a warning.